packages/react-native/ReactCommon/react/renderer/components/drawerlayout/android/react/renderer/components/drawerlayout/AndroidDrawerLayoutState.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+
+#include <react/renderer/core/graphicsConversions.h>
+#include <react/renderer/graphics/Float.h>
+
+#include <folly/dynamic.h>
+#include <react/renderer/mapbuffer/MapBuffer.h>
+#include <react/renderer/mapbuffer/MapBufferBuilder.h>
+
+namespace facebook::react {
+
+/*
+ * State for <AndroidDrawerLayout> component.
+ */
+class AndroidDrawerLayoutState final {
+ public:
+  using Shared = std::shared_ptr<const AndroidDrawerLayoutState>;
+
+  AndroidDrawerLayoutState(){};
+  AndroidDrawerLayoutState(
+      const AndroidDrawerLayoutState& previousState,
+      folly::dynamic data)
+      : drawerWidth((Float)data["drawerWidth"].getDouble()),
+        containerWidth((Float)data["containerWidth"].getDouble()),
+        drawerOnLeft(data["drawerOnLeft"].getBool()),
+        drawerOpened(data["drawerOpened"].getBool()) {};
+
+  const Float drawerWidth = 0;
+  const Float containerWidth = 0;
+  const bool drawerOnLeft = true;
+  const bool drawerOpened = false;
+
+  folly::dynamic getDynamic() const;
+  MapBuffer getMapBuffer() const {
+    return MapBufferBuilder::EMPTY();
+  };
+
+#pragma mark - Getters
```

Comment:
There are no getters so maybe we can remove this mark?

Reply:
Done in 2407494b6357b5120895a0f0cc1c4d165b9b26c0)